### PR TITLE
[VGNWB-340] Cancel edit appellations, multiple appellations highlight

### PR DESCRIPTION
**Pull request by nowke**
Includes #VGNWB-332 as well

* Cancel editing appellation
* Cancel text highlighting by clicking outside
* Fix Relations tab not loading content
* Clicking on relation highlights all appellations associated with it

Related backend PR - https://github.com/diging/vogon-web/pull/193

### CODE REVIEW

**Comment by tjquinn1** [comment]
- When I edit an appellation and don't select text there should be a button that says cancel so I can cancel at that point.
- When I edit an appellation and select new text and then cancel, it cancels but the old annotations does not reappear.
- clicking outside does not deselect text
- the relations tab does work
- clicking on a relation to show connected annotations kind of works. In the template that I tried it shows two out of the three annotations
Screen Shot 2020-07-21 at 3 16 40 PM

**Comment by tjquinn1** [comment]
- Lets move the cancel button inline with other icons. You can make is smaller and just have the text say cancel. Lets also make it red
- cancel highlighting works when I click inside but not outside the text which I think is fine for now.
- When I click on the relation is still does not show me all the appellations related to it. It only shows me the two appellations shown in the relation list and not the third which I think in this case is the object in a subject, object, predicate relationship

**Comment by tjquinn1** [comment]
fix inconsistent indentation in appellation list

```
./src/components/annotator/AppellationListItem.vue?vue&type=template&id=08d49a4f&scoped=true&lang=pug& (./node_modules/cache-loader/dist/cjs.js?{"cacheDirectory":"node_modules/.cache/vue-loader","cacheIdentifier":"8987178e-vue-loader-template"}!./node_modules/vue-loader/lib/loaders/templateLoader.js??vue-loader-options!./node_modules/pug-plain-loader!./node_modules/vuetify-loader/lib/loader.js!./node_modules/cache-loader/dist/cjs.js??ref--0-0!./node_modules/vue-loader/lib??vue-loader-options!./src/components/annotator/AppellationListItem.vue?vue&type=template&id=08d49a4f&scoped=true&lang=pug&)
Module build failed (from ./node_modules/pug-plain-loader/index.js):
Error: /Users/taylorquinn/Desktop/vogon/vogon-web-client/src/components/annotator/AppellationListItem.vue:14:1
    12| 						| This appellation is part of existing relation(s) !!
    13| 					v-btn(small dense @click="editAppellation") Cancel Edit
  > 14| 			v-col(:cols="2" class="text-right")
--------^
    15| =======
    16| 				
    17| 			v-col(:cols="5" class="text-right")

Inconsistent indentation. Expecting either 0 or 4 spaces/tabs.
```

**Comment by nowke** [comment]
Fixed indentations and linting errors. Before running, run `npm install` once.

**Comment by tjquinn1** [comment]
Make it so, Jenkins.

**Comment by tjquinn1** [comment]
Make it so, Jenkins.

**Comment by tjquinn1** [comment]
Make it so, Jenkins.

**Comment by tjquinn1** [comment]
Make it so, Jenkins.

**Comment by tjquinn1** [comment]
Make it so, Jenkins.

File: src/components/annotator/TextDisplay.vue [hunk 1]
```diff
@@ -42,6 +42,11 @@ export default class TextDisplay extends Vue {
 
 	public created() {
 		this.currentAppellations = this.appellations;
+		// store.subscribe((mutation: any, state: any) => {
```
Comment (tjquinn1) [inline src/components/annotator/TextDisplay.vue]: If this isnt needed remove it. Theres no point leaving commented code hanging around

Comment (nowke) [reply src/components/annotator/TextDisplay.vue]: Removed

**Comment by nowke** [comment]
Closing this PR - will implement "Delete appellation" feature with this - https://diging.atlassian.net/browse/VGNWB-253

**Comment by nowke** [comment]
Added Delete appellation functionality

**Comment by tjquinn1** [comment]
Make it so, Jenkins.

**Comment by tjquinn1** [comment]
Make it so, Jenkins.

**Comment by tjquinn1** [comment]
Make it so, Jenkins.

**Comment by tjquinn1** [comment]
Make it so, Jenkins.

**Comment by tjquinn1** [comment]
Make it so, Jenkins.

**Comment by tjquinn1** [comment]
Make it so, Jenkins.

**Pull request merged**